examples/tensorflow/common/export.py

```diff
@@ -26,16 +28,23 @@ def export_model(model: tf.keras.Model, save_path: str, save_format: str) -> Non
     :param save_path: Path to save.
     :param save_format: Model format used to save model.
     """
-
     if save_format == FROZEN_GRAPH_FORMAT:
         input_signature = []
         for item in model.inputs:
             input_signature.append(tf.TensorSpec(item.shape, item.dtype, item.name))
         concrete_function = tf.function(model).get_concrete_function(input_signature)
         frozen_func = convert_variables_to_constants_v2(concrete_function, lower_control_flow=False)
         frozen_graph = frozen_func.graph.as_graph_def(add_shapes=True)
-
         save_dir, name = osp.split(save_path)
         tf.io.write_graph(frozen_graph, save_dir, name, as_text=False)
-    else:
+    elif save_format == SAVED_MODEL_FORMAT:
         model.save(save_path, save_format=save_format)
```

Comment:
@andrey-churkin Did you remember the reason why it was done? https://github.com/openvinotoolkit/nncf/blob/63d0b543bd60cd49061383310ed562a19744004a/nncf/tensorflow/exporter.py#L72